build/widgets/drive/index.js

```diff
@@ -22,14 +22,12 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
  */
-var DriveScanner, DynamicList, Promise, _, async, chalk, driveToChoice, drivelist, getDrives;
+var DriveScanner, DynamicList, Promise, _, chalk, driveToChoice, drivelist, getDrives;
```

Comment:
Line is too long.<br>'Promise' is defined in a future version of JavaScript. Use a different variable name to avoid migration issues.